drivers/shared/executor/executor_test.go

```diff
@@ -59,15 +59,6 @@ var (
 	compute  = topology.Compute()
 )
 
-type testExecCmd struct {
```

Comment:
Note to reviewers: this whole file is marked `!windows` now, so I moved these helpers into `utils_test.go`